### ISSUE TITLE
Timeline tooltip option for 'plain'

### ISSUE THREAD

**Issue by svatoupravdu** [almende/vis#3365]
New feature for displaying styled tooltips is nice, but can be either turned on (to internally styled) or completely off. Some applications already use another tooltip engine (jQuery UI) and would like to stick to that.
My idea is to provide an extra tooltip option which would define whether the tooltip is standard HTML TITLE or is rendered by visjs engine:
```javascript
var options = {
        showTooltips: true,
        tooltip:{
          style:plain|graphic 
        }
}
```
This would give us option to turn tooltips off or on, and if on then whether styled or non-styled.

PS: I'm aware of workaround achievable by styling tooltip to my needs via CSS but that solves just visual part of problem, not behavioural.

**Comment by lewisjb** [comment]
If you are using another 'tooltip engine', then wouldn't you want to set `showTooltips` to `false` altogether?

**Comment by svatoupravdu** [comment]
Nope, because TITLE attribute (which is the data source for another tooltip engine) is ommited completely then and jQueryUI tooltip has nothing to process.
